src/common-flags.ts

```diff
@@ -112,3 +112,9 @@ export const logsDirectory = string({
   description: 'Directory to store logs into',
   env: 'CHE_LOGS'
 })
+
+export const CHE_TELEMETRY = string({
+  char: 't',
```

Comment:
We use this char for `--templates`  and `--access-token`.
So, let's remove it at all.